pkg/webui/console/views/device-overview/index.js

```diff
@@ -67,6 +70,17 @@ class DeviceOverview extends React.Component {
     shouldRedirect: false,
   }
 
+  @bind
+  onExport() {
```

Comment:
The mac state that is fetched for the overview page only contains the `recent_downlinks` and `recent_uplinks`. For this download function to be useful, it is necessary to retrieve the full `mac_state` object. Since this is an on demand function, I'd suggest making a fresh API call here to retrieve the full `mac_state` object and then using this for the download.